src/common/numa.cc

```diff
@@ -137,3 +137,37 @@ int get_numa_node_cpu_set(
   ::close(fd);
   return r;
 }
+
+#elif defined(__FreeBSD__)
+
+int parse_cpu_set_list(const char *s,
+		       size_t *cpu_set_size,
+		       cpu_set_t *cpu_set)
+{
+  return 0;
+}
+
+std::string cpu_set_to_str_list(size_t cpu_set_size,
+				const cpu_set_t *cpu_set)
+{
+  std::string r;
+  r = "0";
+  return r;
+}
+
+std::set<int> cpu_set_to_set(size_t cpu_set_size,
+			     const cpu_set_t *cpu_set)
+{
+  set<int> r;
+  return r;
+}
+
+int get_numa_node_cpu_set(
+  int node,
```

Comment:
please be consistent regarding to the indent. see also https://github.com/ceph/ceph/pull/25920/files#diff-7996880cd1020fda3cee39c2fac542d4R159